examples/fabric-contract-example-gradle/fabric-contract-example-external/src/main/java/org/example/Application.java

```diff
@@ -0,0 +1,39 @@
+/*
+* SPDX-License-Identifier: Apache-2.0
+*/
+
+package org.example;
+
+import java.io.IOException;
+import org.hyperledger.fabric.contract.ContractRouter;
+import org.hyperledger.fabric.shim.ChaincodeServer;
+import org.hyperledger.fabric.shim.ChaincodeServerProperties;
+import org.hyperledger.fabric.shim.NettyChaincodeServer;
+
+public class Application {
+
+    private static final String CHAINCODE_SERVER_PORT = "CHAINCODE_SERVER_PORT";
```

Comment:
I've checked the example at https://github.ibm.com/WHITEMAT/contract-as-a-server/blob/main/java-contract/src/main/java/org/example/ContractBootstrap.java.

It's slightly different - in that in takes the full IP and port number to be consistent with node... Might be worth changing that here as well...

Reply:
Done